### ISSUE TITLE
Duplicate "Layer created by commands" information is reported 

### ISSUE THREAD

**Issue by rnjudge** [tern-tools/tern#942]
**Describe the bug**
When you run Tern on an image that is in the cache, duplicate "Layer created by commands" information is reported.

**To Reproduce**
1. Run tern twice on the same image.
I think this is related to https://github.com/tern-tools/tern/pull/926.

**Error in terminal**
```
Docker image: photon@sha256:fbdae32f534858727fa855af8d548dfa5d98872ef81f466790f7c302a46e8384:
	Layer 1:
		info: Layer created by commands: /bin/sh -c #(nop) ADD file:0d11f538a785fef422a09a6576a7da9c315f0e6b608b8ac67b927038da2514fe in / 
		info: Layer created by commands: /bin/sh -c #(nop) ADD file:0d11f538a785fef422a09a6576a7da9c315f0e6b608b8ac67b927038da2514fe in / 
		info: Found 'VMware Photon OS/Linux' in /etc/os-release.
		info: Retrieved by invoking listing in command_lib/base.yml

```

**Expected behavior**
Should only print once.

**Comment by rnjudge** [comment]
Bumping the priority on this as it will duplicate the line in the report an unlimited number of times depending on how many times you have run tern on the same image. For example, I have been running on the photon image a bunch as I work on and test another issue... here's the output report:

```
ocker image: photon@sha256:fbdae32f534858727fa855af8d548dfa5d98872ef81f466790f7c302a46e8384:
	Layer 1:
		info: Layer created by commands: /bin/sh -c #(nop) ADD file:0d11f538a785fef422a09a6576a7da9c315f0e6b608b8ac67b927038da2514fe in / 
		info: Layer created by commands: /bin/sh -c #(nop) ADD file:0d11f538a785fef422a09a6576a7da9c315f0e6b608b8ac67b927038da2514fe in / 
		info: Layer created by commands: /bin/sh -c #(nop) ADD file:0d11f538a785fef422a09a6576a7da9c315f0e6b608b8ac67b927038da2514fe in / 
		info: Layer created by commands: /bin/sh -c #(nop) ADD file:0d11f538a785fef422a09a6576a7da9c315f0e6b608b8ac67b927038da2514fe in / 
		info: Layer created by commands: /bin/sh -c #(nop) ADD file:0d11f538a785fef422a09a6576a7da9c315f0e6b608b8ac67b927038da2514fe in / 
		info: Layer created by commands: /bin/sh -c #(nop) ADD file:0d11f538a785fef422a09a6576a7da9c315f0e6b608b8ac67b927038da2514fe in / 
		info: Layer created by commands: /bin/sh -c #(nop) ADD file:0d11f538a785fef422a09a6576a7da9c315f0e6b608b8ac67b927038da2514fe in / 
		info: Layer created by commands: /bin/sh -c #(nop) ADD file:0d11f538a785fef422a09a6576a7da9c315f0e6b608b8ac67b927038da2514fe in / 
		info: Layer created by commands: /bin/sh -c #(nop) ADD file:0d11f538a785fef422a09a6576a7da9c315f0e6b608b8ac67b927038da2514fe in / 
		info: Layer created by commands: /bin/sh -c #(nop) ADD file:0d11f538a785fef422a09a6576a7da9c315f0e6b608b8ac67b927038da2514fe in / 
		info: Layer created by commands: /bin/sh -c #(nop) ADD file:0d11f538a785fef422a09a6576a7da9c315f0e6b608b8ac67b927038da2514fe in / 
		info: Layer created by commands: /bin/sh -c #(nop) ADD file:0d11f538a785fef422a09a6576a7da9c315f0e6b608b8ac67b927038da2514fe in / 
		info: Layer created by commands: /bin/sh -c #(nop) ADD file:0d11f538a785fef422a09a6576a7da9c315f0e6b608b8ac67b927038da2514fe in / 
		info: Layer created by commands: /bin/sh -c #(nop) ADD file:0d11f538a785fef422a09a6576a7da9c315f0e6b608b8ac67b927038da2514fe in / 
		info: Layer created by commands: /bin/sh -c #(nop) ADD file:0d11f538a785fef422a09a6576a7da9c315f0e6b608b8ac67b927038da2514fe in / 
		info: Layer created by commands: /bin/sh -c #(nop) ADD file:0d11f538a785fef422a09a6576a7da9c315f0e6b608b8ac67b927038da2514fe in / 
		info: Layer created by commands: /bin/sh -c #(nop) ADD file:0d11f538a785fef422a09a6576a7da9c315f0e6b608b8ac67b927038da2514fe in / 
		info: Layer created by commands: /bin/sh -c #(nop) ADD file:0d11f538a785fef422a09a6576a7da9c315f0e6b608b8ac67b927038da2514fe in / 
		info: Layer created by commands: /bin/sh -c #(nop) ADD file:0d11f538a785fef422a09a6576a7da9c315f0e6b608b8ac67b927038da2514fe in / 

```

**Comment by nishakm** [comment]
@rnjudge I may have a fix for this similar to 728f6193904ea228696affe676fc65acf3217a41
